CHANGELOG_UNRELEASED.md

```diff
@@ -40,6 +40,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - in `finset.v`, new lemmas: `properC`, `properCr`, `properCl`
 - in `ssrnat.v`, new lemmas: `subn_minl`, `subn_maxl`
 - in `ssrnat.v`, new lemma: `oddS`
+- in `finset.v`, new lemmas: `mem_imset_eq`, `mem_imset2_eq`.
+  Ideal names would be `mem_imset` and `mem_imset2`, but these names have just been deprecated from their previous meaning.
```

Comment:
Don't like this idea of temporary name. @maximedenes how coq deals with this kind of situation?

Reply:
This strategy was discussed [here](https://github.com/math-comp/math-comp/wiki/Minutes-June-30-2020] I cannot recall whether @maximedenes attended though...

Reply:
I wasn't there either :wink: still not a fan of temporary names

Reply:
You can say "these names will lose the _eq suffix in the next release", it may be of interest to the user. The reason why is not really helping IMO, and here you are not anyway suggesting we plan to use the ideal names in the future.

Reply:
I don't recall a case like this in Coq.

Reply:
If everybody like temporary names, then @CohenCyril could already make the next pull request for 1.13 so we are sure theses names wil disappear.

Reply:
> Don't like this idea of temporary name. @maximedenes how coq deals with this kind of situation?

I don't like the temporary names either, but I don't see another way without causing a breaking change. And this seems to minor an issue to cause a breaking change. 

> You can say "these names will lose the _eq suffix in the next release", it may be of interest to the user. The reason why is not really helping IMO, and here you are not anyway suggesting we plan to use the ideal names in the future.

I was merely following the formulations used by Cyril in #492 

> If everybody like temporary names, then @CohenCyril could already make the next pull request for 1.13 so we are sure theses names wil disappear.

This is an "after merge" Item, again as in #492

Reply:
It can't be a pull request yet, because the things to be removed aren't there yet ...

Reply:
@chdoc sorry I am reviewing too many pull requests by cyril I thought it was one of his too.

Reply:
> It can't be a pull request yet, because the things to be removed aren't there yet ...

Actually, it could be, I did not think about it, but it just needs to be a pr on top of this one :laughing: